.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```

Comment:
I would prefer we would use `mcr.microsoft.com/devcontainers/go:1.22-bullseye` with features required for woodpecker development like `ghcr.io/devcontainers/features/node:1` with same node version we use in pipelines etc